packages/ra-core/src/form/withDefaultValue.tsx

```diff
@@ -10,7 +10,7 @@ export interface DefaultValueProps extends InputProps {
     initializeForm: typeof initializeFormAction;
 }
 
-export class DefaultValueView extends Component<DefaultValueProps> {
+export class DefaultValueView extends Component<any> {
```

Comment:
Why ?

Reply:
because the `connect` call at the end of the file triggers this cryptic TypeScript error:

```
Argument of type 'typeof DefaultValueView' is not assignable to parameter of type 'ComponentType<never>'.
  Type 'typeof DefaultValueView' is not assignable to type 'ComponentClass<never, any>'.
    Types of property 'propTypes' are incompatible.
      Type '{ decoratedComponent: Requireable<ReactElementLike | ((...args: any[]) => any)>; defaultValue: Requireable<any>; initializeForm: Validator<(...args: any[]) => any>; input: Requireable<object>; source: Requireable<...>; validate: Requireable<...>; }' is not assignable to type 'never'.ts(2345)
```

And I don't know how to silence it.